.github/workflows/tests.yml

```diff
@@ -132,25 +128,25 @@ jobs:
       - uses: actions/setup-python@v4
         with:
           python-version: "3.11"
-          cache: pip
-          cache-dependency-path: requirements-tests.txt
+      - name: Install uv
+        run: curl -LsSf https://astral.sh/uv/install.sh | sh
       - name: Install typeshed test-suite requirements
         # Install these so we can run `get_external_stub_requirements.py`
-        run: pip install -r requirements-tests.txt
+        run: uv pip install -r requirements-tests.txt --system
       - name: Create an isolated venv for testing
-        run: python -m venv .venv
+        run: uv venv .venv
       - name: Install 3rd-party stub dependencies
         run: |
           DEPENDENCIES=$(python tests/get_external_stub_requirements.py)
           if [ -n "$DEPENDENCIES" ]; then
             source .venv/bin/activate
             echo "Installing packages: $DEPENDENCIES"
-            pip install $DEPENDENCIES
+            uv pip install $DEPENDENCIES --system
           fi
       - name: Activate the isolated venv for the rest of the job
         run: echo "$PWD/.venv/bin" >> $GITHUB_PATH
       - name: List 3rd-party stub dependencies installed
-        run: pip freeze --all
+        run: uv pip freeze
```

Comment:
Does uv not do the thing where setuptools etc. are only listed if you pass `--all`?

Reply:
uv doesn't install setuptools/pip/wheel into environments the way the stdlib venv module does, unless you explicitly ask for them with `--seed` (because, the theory goes, you don't need any of them if you have `uv`). So even if uv had a `--all` flag for `uv pip freeze` (it doesn't currently), we wouldn't need to use it

Reply:
And I assume that it does, by default, list setuptools/pip etc if they are installed into an environment. Though I haven't checked.